src/App.js

```diff
@@ -1,64 +1,63 @@
-import React from 'react';
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
 import './App.css';
-import {useState} from 'react';
-import FilterList from './components/FilterList';
+import FilterList from "./components/FilterList";
 
 function App() {
 
-  const [tasks,setTasks] = useState(null);
-  const [status,setStatus] = useState('Pending');
-  const [filter,setFilter] = useState({
-    pending: true,
-    inProgress: true,
-    done: true
-  });
-  const [taskList,setTaskList] = useState([]);
+    const dispatch = useDispatch();
+    const [newTask,setNewTask] = useState(null);
+    const [status,setStatus] = useState('Pending');
```

Comment:
why we use component state `status` ?

Reply:
status is being set from a different input field so we need this state to handle that and pass it to the task object.